mods/doors/init.lua

```diff
@@ -642,8 +646,9 @@ function doors.register_trapdoor(name, def)
 		type = "fixed",
 		fixed = {-0.5, -0.5, -0.5, 0.5, -6/16, 0.5}
 	}
+	
```

Comment:
Please remove the tabulator (2/2 lint warning fixes)

Reply:
I had auto indentation set in my editor. I've turned it off now.